tests/migration/online_migration/yast2_migration.pm

```diff
@@ -217,7 +217,12 @@ sub run {
     }
 
     # migration_sle will show yast2_migration-license-agreement after alt-m
-    assert_screen [qw(yast2-migration-installupdate yast2_migration-license-agreement)], 600;
+    assert_screen [qw(yast2-migration-installupdate yast2_migration-license-agreement yast2_migration-import-gpgkey)], 600;
+    if (match_has_tag 'yast2_migration-import-gpgkey') {
```

Comment:
Please add softfail here for https://bugzilla.suse.com/show_bug.cgi?id=1221444 , maybe later we need remove the workaround if bug confirmed and fixed later.

Reply:
I add softfail to the needle.

Reply:
Ok, anyway, make the test passed firstly.